backend/codesirius/problems/views/reference_solution.py

```
@@ -0,0 +1,177 @@
+            )
+            return CodesiriusAPIResponse(
+                data={"id": reference_solution.id},
+                status_code=status.HTTP_201_CREATED,
+                message="Reference solution created",
+            )
+        logger.warning("Reference solution creation failed due to validation errors")
+        raise ValidationError(serializer.errors)
+
+
+class ReferenceSolutionRetrieveUpdateDestroyAPIView(APIView):
+    permission_classes = [IsProblemOwner]
```

Ruff (0.8.2)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)